### PR TITLE
Update README.md for CPUTemperature

### PR DESCRIPTION
**Pull request by djaus2**
A bit more information with ReadMe making it clear that no hardware is required and does not work on IoT-Core.

- If PR is fixing any issue please add `Fixes #1234` as a first thing in the description of your PR
- Describe what is the issue being fixed
- If there is any follow-up work please create issues for that
- If your PR is adding device binding please make sure to read [conventions for devices APIs](https://github.com/dotnet/iot/blob/master/Documentation/Devices-conventions.md)
- Avoid force pushing to your PR (especially on large PRs) - it makes it much harder to incrementally review

### CODE REVIEW

File: src/devices/CpuTemperature/README.md [hunk 1]
```diff
@@ -5,6 +5,11 @@ Device bindings for the CPU Temperature Sensor
 ## Summary
 
 Returns the current temperature of the CPU Temperature Sensor. Useful telemetry in its own right, but also useful for calibrating the Raspberry Pi Sense HAT.
+  
+Note: No hardware needs to connected for this to work. It gets the CPU core temperature, typically about 40 degress Celcius.
```
Comment (joperezr) [inline src/devices/CpuTemperature/README.md:9]:
```suggestion
Note: No hardware needs to be connected in order for this binding to work. The way it works is that it gets the CPU core temperature, which is typically around 40 degrees Celsius.  
```

File: src/devices/CpuTemperature/README.md [hunk 2]
```diff
@@ -5,6 +5,11 @@ Device bindings for the CPU Temperature Sensor
 ## Summary
 
 Returns the current temperature of the CPU Temperature Sensor. Useful telemetry in its own right, but also useful for calibrating the Raspberry Pi Sense HAT.
+  
+Note: No hardware needs to connected for this to work. It gets the CPU core temperature, typically about 40 degress Celcius.  
+On a Raspberry Pi. it does not work with Windows 10 IoT-Core; Raspian is the required operating system.
```
Comment (joperezr) [inline src/devices/CpuTemperature/README.md:10]:
```suggestion
The binding requires a Raspberry Pi running Raspbian. It won't work if the Raspberry Pi's OS is Windows 10 IoT-Core. 
```

File: src/devices/CpuTemperature/README.md [hunk 3]
```diff
@@ -5,6 +5,11 @@ Device bindings for the CPU Temperature Sensor
 ## Summary
 
 Returns the current temperature of the CPU Temperature Sensor. Useful telemetry in its own right, but also useful for calibrating the Raspberry Pi Sense HAT.
+  
+Note: No hardware needs to connected for this to work. It gets the CPU core temperature, typically about 40 degress Celcius.  
+On a Raspberry Pi. it does not work with Windows 10 IoT-Core; Raspian is the required operating system. 
+
+Same as ```/opt/vc/bin/vcgencmd measure_temp```
```
Comment (joperezr) [inline src/devices/CpuTemperature/README.md:12]:
```suggestion
```

**Comment by joperezr** [comment]
Thanks for the contribution! Added some comments for rewording a few things. Let me know if you agree and if so let's commit them and merge this.

File: src/devices/CpuTemperature/README.md (hunk 2 above)
Comment (krwq) [reply src/devices/CpuTemperature/README.md:10]: The binding requires any Linux (possibly Unix will work as well), Raspberry PI is not required.

**Comment by krwq** [comment]
@djaus2 had a chance to see @joperezr suggestion? If no action I'll submit Jose's suggestions and merge

**Comment by djaus2** [comment]
Ok lets close this.
Thanks

**Comment by djaus2** [comment]
I mean lest commit this.

**Comment by krwq** [comment]
@djaus2 / @joperezr can you click submit suggestion button? I don't seem to be able to

**Comment by djaus2** [comment]
I might of taken the wrong action, please check.

**Comment by krwq** [comment]
@djaus2 I think you closed PR instead without making any changes. What I meant is to check @joperezr's comment and there should be a button to submit the suggestion to this PR (or just update the PR), after that LGTM

**Pull request closed without merging**